shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -634,6 +634,12 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
     super.onCreate(savedInstanceState);
 
+    if (savedInstanceState != null
+        && savedInstanceState.getBoolean(
+            FlutterActivityAndFragmentDelegate.ON_BACK_CALLBACK_ENABLED_KEY)) {
```

Comment:
Are you following an existing pattern in our code? If not then I suggest having this key hold if callback is enabled. That way no key indicates "default" and a true or false indicate a preference.

Reply:
Just to be clear, do you mean replacing with something like
```java
if (savedInstanceState != null) {
    Boolean frameworkHandlesBack = savedInstanceState.getBoolean(
            FlutterActivityAndFragmentDelegate.ON_BACK_CALLBACK_ENABLED_KEY));
    setFrameworkHandlesBack(frameworkHandlesBack);
}
```

Reply:
yes

Reply:
done!